lib/config/generator.js

```diff
@@ -203,19 +201,56 @@ function getChecker() {
 
 /**
  * @private
- * @param  {String[]} violatedRuleNames
+ * @param  {Object} errors
  * @return {Object[]}
  */
-function generateRuleHandlingPrompts(violatedRuleNames) {
+function generateRuleHandlingPrompts(errors) {
+    // Generate list of rule names, sorted by violation count (descending)
+    var violatedRuleNames = Object.keys(errors);
+    violatedRuleNames.sort(function(a, b) {
+        return errors[b].violations - errors[a].violations;
+    });
+
     return violatedRuleNames.map(function(ruleName) {
+        var violationCount = errors[ruleName].violations;
+        var fileCount = Object.keys(errors[ruleName].files).length;
         var prompt = assign({}, prompts[1]);
-        prompt.name = colors.green(ruleName) + ': ' + prompt.name;
+
+        prompt.associatedRuleName = ruleName;
```

Comment:
This is okay, but only because the prompt module doesn't have a meaning associated with the `associatedRuleName` key. The `prompt` object was intended to be an object of settings specifically for the `prompt` module.